src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -8,24 +8,24 @@
                     placeholder="Add a comment here...">
                 </textarea>
             </div>
-            <div *ngFor='let comment of workItem.relationalData.comments; let counter = index' class="comments-wrap">  
+            <div *ngFor='let comment of workItem.relationalData?.comments?.slice().reverse()' class="comments-wrap">
                 <div>
                     <div class="user-avatar pull-left">
-                        <img id="{{'comment_avatar_' + counter}}"
-                        class="user-assign-avatar pull-left" 
+                        <img class="user-assign-avatar pull-left"
                         tooltipPlacement="right"
                         tooltip="{{comment.relationalData.creator?.attributes?.fullName}}"
                         *ngIf="comment.relationalData.creator?.attributes?.imageURL" 
                         [src]="comment.relationalData.creator?.attributes?.imageURL | almAvatarSize:30"
                         />
                     </div>
-                    <small class="pull-right createTimeBlock" id="{{'comment_time_' + counter}}">{{comment.attributes['created-at'] | date:'medium'}}</small>
```

Comment:
Why did you remove the ID ? This is for functional testing. @naina-verma Please confirm.